__tests__/utils.test.js

```diff
@@ -102,3 +105,16 @@ describe("formatComments", () => {
     expect(formattedComments[0].created_at).toEqual(new Date(timestamp));
   });
 });
+
+describe('checkarticleIdExists', () => {
+  it('returns a value of true if article id is assigned to an article', () => {
+    return checkArticleIdExists(1).then((result) => {
+      expect(result).toBe(true)
+    })
+  })
+  it('returns a value of false if article id is not assigned to an article', () => {
+    return checkArticleIdExists(99).then((result) => {
+      expect(result).toBe(false)
+    })
+  })
```

Comment:
Do you need to seed the database as the util function uses it? Despite the fact that it's not a destructive query, a query is still happening to a database that is also being tested in another file _with_ potentially destructive queries. Right now, not reseeding the database before each test in your util test file is ok, but eventually, it might cause a problem - you might end up failing these tests because of another test in your api test file :)